src/style.css

```
@@ -1,3 +1,84 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  background-color: gray;
+}
+
+html {
+  --color-active: #00acdc;
+}
+
+.header {
+  display: flex;
+  background: #fff;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  justify-content: flex-end;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 0;
+}
+
+/* a {
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+} */
+
+.header li {
```

```suggestion
.nav_item {
```
style by classes instead by tags. exclusion can be html, body or reset default styles for some tags.
better to add class `.nav_item` for `li` and `.nav_link` for `a` tag.